### PR TITLE
Update cert-manager docs to match new template #4552

### LINKED ISSUE #4552: Update cert-manager package documentation
## Feature Request

There is an effort to update and standardize package documentation. Update the latest cert-manager package (1.8.0) with this updated template.

### PR DESCRIPTION
**Pull request by seemiller**
## What this PR does / why we need it

This pull request updates the cert-manager documentation to follow an updated template for package documentation.

## Which issue(s) this PR fixes

Fixes: #4552

## Describe testing done for PR

n/a

## Special notes for your reviewer

n/a

### CODE REVIEW

**Comment by seemiller** [comment]
@LappleApple Could you take a look at the updated cert-manager docs? Thanks!

**Review by lasomethingsomething — COMMENTED** [review]
Thanks for doing this @seemiller, only one comment about Lines 35-38

File: addons/packages/cert-manager/1.8.0/README.md [hunk 1]
```diff
@@ -1,60 +1,83 @@
...
 
 | Config | Values | Default | Description |
 |--------|--------| -----------|-------------|
 | `namespace` | Any valid namespace | `cert-manager` | Optional. The namespace in which to deploy cert-manager. |
 
-### Installation
+#### <Value>
 
-The cert-manger package can be installed with the Tanzu package CLI.
+Description: include a description of why you would use this native configuration option and reference to configurable options in application documentation
```
[the remark this answers is not in the document]
Comment (seemiller) [reply addons/packages/cert-manager/1.8.0/README.md]: Deleted!

**Review by davidvonthenen — APPROVED** [review]

**Comment by dvonthenen** [comment]
LGTM but probably not helpful since I'm not in any of those groups required for approval

**Pull request merged**